package.json

```diff
@@ -65,6 +65,14 @@
                     "command": "vscode-docker.containers.composeGroup.logs",
                     "when": "config.docker.containers.groupBy == 'Compose Project Name'"
                 },
+                {
+                    "command": "vscode-docker.containers.composeGroup.start",
+                    "when": "config.docker.containers.groupBy == 'Compose Project Name'"
+                },
+                {
+                    "command": "vscode-docker.containers.composeGroup.stop",
+                    "when": "config.docker.containers.groupBy == 'Compose Project Name'"
+                },
                 {
                     "command": "vscode-docker.containers.composeGroup.restart",
```

Comment:
Restart and down overlap with commands that are executed on compose files--not tree items--so we'll continue to never show those in the command palette.